docs/infrastructure/microk8s-quickstart.md

```diff
@@ -0,0 +1,348 @@
+
+#### 2. External Databases
+
+If you have decided to go with the [default values for the installation](####1.Default-Values), your microk8s cluster will contain a postgresql database within itself. This is not recommended as the data stored in the database will not be as durable if you store it in an external database.
+To use an external database, you should follow the following steps:
+
+1. Have a postgres 9 or postgres 10 database cluster
+2. The database cluster must have the following databases
+   1. defaultdb
+   2. accounts
+   3. analysis
+   4. results
+   5. metrics
+   6. filestore
+   7. jobs
+3. In the values file that you provide during the `helm` installation, define the following for the databases described above:
+   1. `global.<database>db.create=false` - disable the creation of this database inside the cluster.
+   2. `global.<database>db.postgresqlUsername` - the username that will have access to this database.
+   3. `global.<database>db.postgresqlPassword` - the passworf for the given user.
+   4. `global.<database>db.host` - hostname of the external database.
+4. The database cluster must have the following databases
+   1. activities
+   2. hotspots
+   3. listener
+5. In the values file that you provide during the `helm` installation, define the following for the databases described above:
+   1. `<database>db.create=false` - disable the creation of this database inside the cluster.
+   2. `<database>db.postgresqlUsername` - the username that will have access to this database.
+   3. `<database>db.postgresqlPassword` - the passworf for the given user.
+   4. `<database>db.host` - hostname of the external database.
+6. Note that the provided user must be the `owner` of the database(s):
+    >ALTER ROLE \<user> WITH CREATEDB;
+
+    Then you can connect as that user and create the databases:
+    ```sql
+    CREATE DATABASE accounts WITH OWNER=<user>;
+    CREATE DATABASE analysis WITH OWNER=<user>;
+    CREATE DATABASE results WITH OWNER=<user>;
+    CREATE DATABASE metrics WITH OWNER=<user>;
+    CREATE DATABASE filestore WITH OWNER=<user>;
+    CREATE DATABASE jobs WITH OWNER=<user>;
+    CREATE DATABASE activities WITH OWNER=<user>;
+    CREATE DATABASE hotspots WITH OWNER=<user>;
+    CREATE DATABASE listener WITH OWNER=<user>;
+    ```
+
+## 6. Accessing the Codacy UI
+
+In the previous section, you should have defined values for `codacy-api.ingress.hosts.host`, `global.codacy.url`, and `global.codacy.backendUrl`. This value will be the address where you can access the Codacy UI from your web browser.
+
+## 7. Troubleshooting
+
+### 1. AWS EC2 specific issues
```

Comment:
I think this may be quite specific to the test that you did and may confuse customers since we also provide instructions to install Codacy on Amazon EKS.

I was thinking that customers using microk8s will use any kind of infrastructure, most probably one that is managed by them?

Consider removing this section.

Reply:
👍 removing